org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/wizard/template/ProjectFile.java

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2017 itemis AG (http://www.itemis.de) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.ui.wizard.template;
+
+/**
+ * Part of a ProjectTemplate to define files the new project wizard is creating when a specific template is selected. Instantiated inside
+ * methods from classes annotated with {@link ProjectTemplate}.
+ * 
+ * @author Arne Deutsch - Initial contribution and API
+ * @since 2.14
+ */
+public final class ProjectFile {
```

Comment:
final modifiers are typically not used in Xtext to allow customizing.

Reply:
Will open up the API (see above).

Reply:
The type is still final. Is that intentional?

Reply:
The class is not used any more. Will remove it with next comit.